### DISCUSSION TITLE
Inference on mac os 10.14

### DISCUSSION THREAD

**Discussion by xidiancpy** [microsoft/onnxruntime#5062] [Ideas / Feature Requests]
**Describe the bug**
1. Compile when I just load the header file ”#include <onnxruntime_cxx_api.h>“ in main.cpp, Will appear 
Undefined symbols for architecture x86_64: 
  "_OrtGetApiBase", referenced from: 
      ___cxx_global_var_init in main.o 
ld: symbol(s) not found for architecture x86_64 
clang: error: linker command failed with exit code 1 (use -v to see invocation)Undefined symbols for architecture x86_64: 
  "_OrtGetApiBase", referenced from: 
      ___cxx_global_var_init in main.o 
ld: symbol(s) not found for architecture x86_64 
clang: error: linker command failed with exit code 1 (use -v to see invocation)

2. Compile when I just load the lib file ”libonnxruntime.1.3.0.dylib“ in xcode "Link Binary With Libraries", Will appear 

dyld: Library not loaded: @rpath/libonnxruntime.1.3.0.dylib 
  Referenced from: /Users/quanshi/Desktop/pengyu/ProtraitSeg/DerivedData/ProtraitSeg/Build/Products/Release/ProtraitSeg 
  Reason: image not found 

**Urgency**
I use the same library and code in windows and Linux are not able to run normally, I hope to get a reply within three days.

**System information**
- OS Platform and Distribution (e.g., Linux Ubuntu 16.04):
- ONNX Runtime installed from (source or binary):https://github.com/microsoft/onnxruntime/releases
- ONNX Runtime version:1.3.0/1.4.0
- GCC/Compiler version (if compiling from source):xcode 11

**To Reproduce**
#include <assert.h>
#include <onnxruntime_cxx_api.h>
#include "opencv2/core/core.hpp"
#include "opencv2/imgproc/imgproc.hpp"
#include "opencv2/highgui/highgui.hpp"
#include <opencv2/imgproc/types_c.h>
#include <opencv2/photo.hpp>
#include <iostream>

using namespace std;
using namespace cv;
#define INPUT_SIZE 256
#define TIME_TEST 1
int main(int argc, char* argv[]) {

	float scale = 1/255.0;
	Scalar mean = Scalar(104.0, 112.0, 121.0);
	Size dstSize(INPUT_SIZE, INPUT_SIZE);
	VideoCapture cap;
	int origin_w;
	int origin_h;
	Mat ori_img, ori_to_net, blob, alpha, sum, alpha_con, img_gray, dst_color, pre_img, dst, alpha_result, src_gray, detected_edges;
	alpha = Mat::zeros(INPUT_SIZE, INPUT_SIZE, CV_32FC1);
	blob = Mat::zeros(INPUT_SIZE, INPUT_SIZE, CV_32FC3);
	// default: capture video from camera  
	cap.open(0);
	if (!cap.isOpened())
	{
		printf("\nCan not open camera or video file\n");
		system("PAUSE");
		return -1;
	}
	while (1)
	{
		cap >> ori_img;
		if (ori_img.empty())
		{
			break;
		}
		origin_w = ori_img.cols;
		origin_h = ori_img.rows;

#ifdef TIME_TEST
		double time = 0;
		double counts = 0;
		LARGE_INTEGER nFreq;
		LARGE_INTEGER nBeginTime;
		LARGE_INTEGER nEndTime;
		QueryPerformanceFrequency(&nFreq);
		QueryPerformanceCounter(&nBeginTime);
#endif
		imshow("alpha", ori_img);
		char c = (char)waitKey(1);
		if (c == 27)   // quit after ESC
			break;
	}
	cap.release();
	alpha.release();
	blob.release();
	return 0;
}

**Expected behavior**
I want to know if additional files need to be installed under Mac OS? Please help me complete the compilation.

**Comment by faxu** [comment]
Sorry for the delay; were you able to figure this out or still need assistance?
